draft-ietf-tls-tls13.md

```diff
@@ -4008,7 +4023,8 @@ is listed below:
    1.3" column with the following four values: "Client", indicating
    that the server shall not send them. "Clear", indicating
    that they shall be in the ServerHello. "Encrypted", indicating that
-   they shall be in the EncryptedExtensions block, and "No" indicating
+   they shall be in the EncryptedExtensions block, "Certificate" indicating that
+   they shall be in the Certificate block, and "No" indicating
```

Comment:
Might be worth a parenthetical that the Certificate message contents are also encrypted.